File: java/Arunsir/factorial.java
import java.util.Scanner;

public class factorial {
    public static void main(String[] args) {
        System.out.println("Enter number for factorial.");
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int fact = 1;
        for (int i = 1; i <= a; i++) {
            fact *= i;

        }
        System.out.println("Factorial is = " + fact);
    }
}

File: c language/suyash sir/factorial.c
//Write a program to read a number and display its factorial
#include <stdio.h>
int main(){
long fact=1;
int a;
printf("Enter any number:\n");
scanf("%d",&a);

for (int i = 1; i <=a; i++)
{
    fact *=i;
}
printf("factorial = %ld",fact);
    return 0;
}